src/site/content/en/blog/quintoandar/index.md

```diff
@@ -0,0 +1,225 @@
+---
+title: How QuintoAndar increased conversion rates and pages per session by improving page performances
+subhead: A project focused on optimizing Core Web Vitals and migrating to Next.js resulted in a 5% increase in conversion rates and a 87% increase in pages per session.
+authors:
+  - jlwagner
+date: 2021-12-06
+hero: image/vgdbNJBYHma2o62ZqYmcnkq3j0o1/PHqBoMdg3WgcQWWIIkWj.jpg
```

Comment:
TODO: We need a 3200x960 image for the hero. The hero image in the doc wasn't sufficient in terms of dimensions.

cc: @danielasy